wp-content/themes/currentorg/inc/business-directory-plugin.php

```diff
@@ -0,0 +1,186 @@
+<?php
+/**
+ * Functions modifying the Business Directory Plugin
+ *
+ * @link https://github.com/INN/umbrella-currentorg/pull/24
+ */
+
+/**
+ * Add custom form fields to wpbdp_tag taxonomy add/edit pages
+ */
+function wpbdp_tag_edit_form_fields( $tag ){
+
+	if( $tag ){
+
+		$wpbdp_tag_meta = get_term_meta($tag->term_id);
+		$wpbdp_tag_parent_category = $wpbdp_tag_meta['wpbdp_tag_parent_category'][0];
+		$wpbdp_tag_parent_category = str_replace('wpbdp_category--', '', $wpbdp_tag_parent_category);
+
+	} else {
+
+		$wpbdp_tag_parent_category = '';
+
+	}
+
+	printf( 
+		'<tr class="form-field">
+			<th scope="row" valign="top">
+				<label for="wpbdp_tag_parent_category">%1$s</label>
+			</th>
+			<td>
+				%2$s
+				<p class="description">%3$s</p>
+			</td>
+		</tr>',
+		__( 'Parent Category' ),
+		wp_dropdown_categories(
+			array(
+				'taxonomy'         => 'wpbdp_category',
+				'id'               => 'wpbdp_tag_parent_category',
+				'name'             => 'wpbdp_tag_parent_category',
+				'show_option_none' => __( 'Select category' ),
+				'depth'            => 1,
+				'echo'			   => false,
+				'selected'		   => $wpbdp_tag_parent_category
+			)
+		),
+		__( 'Select the parent category for this tag.' )
+	);
+
+}
+add_action('wpbdp_tag_add_form_fields', 'wpbdp_tag_edit_form_fields');
+add_action('wpbdp_tag_edit_form_fields', 'wpbdp_tag_edit_form_fields');
+
+/**
+ * Save custom form fields from wpbdp_tag taxonomy add/edit pages
+ */
+function wpbdp_tag_form_fields_save( $term_id, $tt_id ) {
+
+    if ( !empty( $_POST['wpbdp_tag_parent_category'] ) ) {
```

Comment:
Somewhere in here we need to verify the nonce for this form submission and make sure that the `$_POST['wpbdp_tag_parent_category']` value here is what we expect it to be.

Reply:
@benlk I went ahead and added a nonce verification to this in commit https://github.com/INN/umbrella-currentorg/pull/24/commits/1f6a699c63eb9be2ba6396fa863e296f61c44f4c.